newIDE/app/src/MainFrame/EditorContainers/HomePage/BuildSection/MaxProjectCountAlertMessage.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import * as React from 'react';
+import { Trans } from '@lingui/macro';
+
+import Text from '../../../../UI/Text';
+import RaisedButton from '../../../../UI/RaisedButton';
+import AlertMessage from '../../../../UI/AlertMessage';
+import { Line, Column } from '../../../../UI/Grid';
+import { type Limits } from '../../../../Utils/GDevelopServices/Usage';
+
+type Props = {|
+  onUpgrade: () => void,
+  limits: Limits,
+|};
+
+export const MaxProjectCountAlertMessage = ({ onUpgrade, limits }: Props) => (
+  <Line>
+    <Column expand>
+      <AlertMessage
+        kind="warning"
+        renderRightButton={() => (
+          <RaisedButton
+            primary
+            label={<Trans>Check our premiums plans</Trans>}
+            onClick={onUpgrade}
+          />
+        )}
+      >
+        <Text size="block-title">
+          <Trans>
+            You've reached your maximum storage of{' '}
+            {limits.capabilities.cloudProjects.maximumCount}
+            cloud-based projects
+          </Trans>
+        </Text>
+        <Text>
+          <Trans>
+            Update to GDevelop Premium to get more storage, one click
```

Comment:
Should it be upgrade?

Reply:
And we should maybe change the message depending if the user has no subscription (check out our premium plans), an indie (consider upgrading to pro to get more storage) or a pro subscription (damn, too many projects, delete some).